package.json

```diff
@@ -157,12 +158,13 @@
     "eslint-plugin-vue": "5.1.0",
     "faker": "4.1.0",
     "husky": "1.3.1",
-    "jest": "24.1.0",
+    "jest": "24.7.1",
     "jest-serializer-html": "6.0.0",
+    "jquery": "3.4.0",
```

Comment:
Moving `jquery` to `devDependencies` as we bundle it in the build so doesn't need to be in the artefact after that.